src/videojs-http-streaming.js

```diff
@@ -1127,6 +1127,7 @@ class VhsHandler extends Component {
 
     this.player_.tech_.on('keystatuschange', (e) => {
       if (e.status !== 'output-restricted') {
+        this.playlistController_.updatePlaylistByKeyStatus(e.keyId, e.status);
```

Comment:
Do we want to keep all the code listed further for cases when we receive `output-restricted`?

Reply:
I guess it is not longer valid

Reply:
Yeah I wasn't sure the best approach here. Leaving the code below, we'll maintain the same behavior as before where we exclude all the HD renditions on a single `output-restricted` status. I was also originally considering having this behind an options flag? Definitely open to some discussion on what we ultimately should have as the default behavior here.

Reply:
as far as I understand it conflicts with the proposed, since when we receive `output-restricted` we will exclude every HD+ rendition.

Reply:
I guess we should just remove it

Reply:
I mean the change is straightforward and should just work, do we need a flag for this?

Reply:
Yeah, I don't really see much downside in keeping this as is without a flag as the playlists with a keyId and no usable keystatus would not be playable anyway.  Also agree, I think the HD rendition filtering is safe to remove here as the same thing would be accomplished through the new logic. Removing the rest below and simplifying this listener.